_posts/2017-08-10-rubyonrails.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Learn Rails and build web apps"
```

Comment:
Wikipedia uses "web apps" while google seems to use "Web Apps" ( [ref](https://developers.google.com/web/progressive-web-apps) ).
So I'm changing it to "Web Apps".